src/main/resources/db/migration/V002__init.sql

```diff
@@ -13,4 +13,9 @@ CREATE TABLE IF NOT EXISTS email_templates (
 
 
 INSERT INTO email_templates (created_on, last_modified, locale, subject, to_email, from_email, content, type)
-  VALUES (CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, 'en_US', 'Registration Activation Link', NULL, NULL, '<a href=http://localhost:8080/api/account/v1/activate/{token}>Activate Account Confirmation Link', 'REGISTRATION');
+  VALUES (CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, 'en_US', 'Registration Activation Link', NULL, NULL, '<a href=http://localhost:3000/activating/{token}>Activate Account Confirmation Link', 'ACTIVATING');
```

Comment:
Shouldn't insert test data to git. This means deploying the code to production would have this `localhost ` thing as well

Reply:
Need clarify